azure_li_services/schema.py

```diff
@@ -0,0 +1,141 @@
+schema = {
+    'version': {
+        'required': True,
+        'type': 'date'
+    },
+    'instance_type': {
+        'required': False,
```

Comment:
No instance type defaults to `LargeInstance`? Why not require this to avoid ambiguity?

Reply:
if not given the default is LargeInstance by the code. I have no problem to make this a requirement.